addons/hr_expense/models/hr_expense.py

```diff
@@ -471,6 +465,21 @@ def _add_followers(self):
         users = self._get_users_to_subscribe()
         self.message_subscribe_users(user_ids=users.ids)
 
+    @api.model
+    def _update_follower_values_on_create(self, values):
+        # Add the followers at creation, so they can be notified
+        employee_id = values.get('employee_id')
+        if not employee_id:
+            return
+
+        employee = self.env['hr.employee'].browse(employee_id)
+        users = self._get_users_to_subscribe(employee=employee) - self.env.user
+        message_follower_ids_values = []
```

Comment:
the postfix `_values` is useless and confusing. Use `message_follower_ids` or even `values['message_follower_ids']` (there should not be many partners to loop on, typically one.)

Reply:
```message_follower_ids_values``` doesn't only contains a list of ids, that's why I suffixed this variable with ```_values```